src/app/beer_garden/config.py

```diff
@@ -811,7 +769,7 @@ def _parse_args(args: Sequence[str]) -> Tuple[YapconfSpec, dict]:
                     "items": {
                         "enable": {
                             "type": "bool",
-                            "default": True,
+                            "default": False,
                             "description": "Run an HTTP server",
                         },
```

Comment:
Could you also change the description for this in your PR. `Run an HTTP server` is not accurate. This is to publish events to an HTTP server.